src/app/controllers/users/create.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
+flnd.userCreate = {
+  configureSubmit: function(config, $scope, messenger, authService, $uibModalInstance) {
+    return function() {
+        $scope.loading = true;
+
+        authService.post(config.users, $scope.user)
+            .then(function success() {
+
+                $uibModalInstance.close();
+
+            }, function error(response) {
+
+                messenger.displayErrorResponse($scope, response);
```

Comment:
However, when an error occurs, the modal popup needs to remain open and display the error message.